src/compas_rhino/geometry/curves/nurbs.py

```diff
@@ -0,0 +1,527 @@
+from itertools import groupby
+from compas.geometry import Point
+# from compas.geometry import Vector
+# from compas.geometry import Transformation
+# from compas.geometry import Frame
+# from compas.geometry import Circle
+# from compas.geometry import Box
+from compas.utilities import linspace
+
+try:
+    from compas.geometry import NurbsCurve
```

Comment:
Again, unclear to me why this can fail, i think it deserves a comment, because it's not evident.